### PR TITLE
The Audio Update Pt 1: Second Coming

### PR DESCRIPTION
**Pull request by nauticall**
## About The Pull Request
See https://gitlab.com/cmdevs/colonial-warfare/-/merge_requests/1688 for the original thing as well as sound samples.

The culmination of like 20 combined hours of combing the A:CM sound library and cherrypicking sounds to implement into CM. The files were all numbered (not labeled) so it was an absolute pain to sort through everything and see which ones fit where.

Some of the sounds may be controversial, so let me know if anything seems better off as it was old.\

Per developer and community request this version does not come with a bloated changelog, extra voice lines, that beefy hijack thing, or that funky acid sound.

## Why It's Good For The Game

This PR aims to bring a fresh coat of paint to the sounds you've spent years listening to over and over again. More stuff has more bass, no longer sounds like a watermelon being slapped with a stick, and overall gives better sound quality, hopefully leading to more immersion.

Hopefully.

I intend for this PR to be testmerged so that people can judge what sounds they like, what sounds they don't like, and what they'd like added.

In the future I'd like to expand on even more sounds, but this PR is a bit bloated as-is already.

## Changelog

:cl:
add: Added more than 150 new sounds into the game; most were ported from A:CM and stock SS13.
add: New firing sounds for the M37, Mk221/Mk210, M4A3, and 88 Mod 4.
add: New sounds for shotgun shell loading, glass breaking, explosions, and explosions greater than 300 power, e.g. OBs and max caps.
add: Added various new interaction noises, particularly when damaging barricades, prying doors, and picking up/dropping items.
add: Several tools now have pickup and drop sounds. In addition to the toolbelt tools, these also include binoculars and fire extinguishers.
/:cl:

### CODE REVIEW

**Comment by Carnage068** [comment]
OH GOD PLEASE NO NOT THIS WHY

**Comment by stanalbatross** [comment]
> OH GOD PLEASE NO NOT THIS WHY

Please give actual feedback

**Comment by JohnDoe50974** [comment]
Some preliminary things I've noticed and opinions:
I've probably missed some, but all the replacements for existing SFX SGTM, my personal favourite are the glass breaking sounds, very crunchy and satisfying.
Some of the sounds are missing, however:

- I've noticed a book pick up and drop SFX, yet nothing happens when I pick up or drop a book
- Were your "keyboard_1/2/3.ogg" files meant to replace or supplement existing keyboard typing sounds? Because neither now plays when clicking on any console.
- The generic un/holster SFX will need to be made a tiny bit quieter, and will likely need to have some kind of a delay to it to work, because it's incredibly easy to spam and sounds grating when spammed. (See: helmet knocking noise addition incidents). If not possible, we'd be better off removing that sound effect altogether.
- Rest of the pick up and drop sounds are fine. It's pretty interesting to give certain items pick up and drop SFX like this. Looking forward to seeing this tested!

**Comment by Carnage068** [comment]
> > OH GOD PLEASE NO NOT THIS WHY
> 
> Please give actual feedback

I did. 10 months ago. Took about 15-20 minutes out of my day to listen to the audio files they posted, gave my opinion on them, made a suggestion (or a heavy recommendation I guess). No response or reaction whatsoever. Not really sure what's expected of me this time around, considering everything I wanted to say has already been said before. I guess I could have just ignored the PR, but I wanted to express my dissent without taking too much time out of my day.

**Comment by neroisvcute** [comment]
New sounds is always welcome to me, especially audible emotes for marines and xenos. Can’t listen right now but would love to play test this.

File: code/game/objects/items/devices/defibrillator.dm [hunk 1]
```diff
@@ -77,11 +77,11 @@
 	ready = !ready
 	user.visible_message(SPAN_NOTICE("[user] turns [src] [ready? "on and takes the paddles out" : "off and puts the paddles back in"]."),
 	SPAN_NOTICE("You turn [src] [ready? "on and take the paddles out" : "off and put the paddles back in"]."))
+	playsound(get_turf(src), "sparks", 25, 1, 0)
 	if(ready)
-		w_class = SIZE_LARGE
```
Comment (fira) [inline code/game/objects/items/devices/defibrillator.dm:81]: Cough Cough think this reverses paddles change, please add it to changelog if you really want it in this (personally don't think it's related to sound)

**Review by fira — COMMENTED** [review]
Aside from the paddles **\*cough\*** oddity **\*cough\***, looks good to me

Discussed with Walter & spooky regarding memory footprint of the variables on /obj/item, conclusion being it's fine in this usage

**Comment by TopHatPenguin** [comment]
**My thoughts so far during the test merge:**

- The paper sound is an absolute delight to listen to.
- Regular computers no longer make a sound when you click on them at all. (bug)


will edit with more as I notice stuff

**Comment by PurpleCIoud** [comment]
Oh boy, the spam of sounds.
ways to fix- 
Make it a lot quieter
Make it happen once in a while not all the time

**Comment by lcarapaica** [comment]
It's really refreshing but a few observations:
-Tools (which some sounds are somewhat questionable, like the screwdriver one) and the drawing out your weapon sound is awfully loud for how often they get used.
-That round I noticed a bug with armors and uniforms, if you grab and promptly drop an USCM uniform, it strips you of your armor, might not be related but reporting it here.
Overall pretty nice tho.

**Comment by morrowwolf** [comment]
I'm stick getting new warcry lines if that's not intentional.

If it is, I'm not a big fan of them.

**Comment by nauticall** [comment]
> I'm still getting new warcry lines if that's not intentional.
> 
> If it is, I'm not a big fan of them.

Yeah, those aren't intentional, sorry. They'll be removed in the next test.

**Comment by morrowwolf** [comment]
> > I'm still getting new warcry lines if that's not intentional.
> > If it is, I'm not a big fan of them.
> 
> Yeah, those aren't intentional, sorry. They'll be removed in the next test.

I will say everything else sounds really good!  This is fantastic work.

**Comment by Diegoflores31** [comment]
Very soulfull i really like the equip sounds . But it would be neat to make explosions a bit louder or a least the bigger ones since OB sounds kinda lame rn

**Comment by Carnage068** [comment]
> Addressing some feedback here
> 
> * The new Marine voice lines that were included in the PR before are (or should be) completely gone. There's no need to put feedback on something that doesn't exist anymore. The needhelps I can address but overall the latter are the only new "voice lines" that are included in this version.
> * The volumes and frequencies of the sounds are noted and will be adjusted for a future test.
> * Keyboard sound and defib bugs are noted.
> 
> I'm actually surprised nobody commented on the M37 shotgun sound despite me being told that it would be "highly controversial". I guess people like it?

Oh, well in that case, I'm actually all for it. All the other sounds I listened to sound pretty good, honestly- It was just the warcries and that one needhelp that I didn't like. Here's hoping this gets merged!

**Pull request closed without merging**